src/libstd/sync/condvar.rs

```diff
@@ -219,6 +219,61 @@ impl Condvar {
         }
     }
 
+    /// Blocks the current thread until this condition variable receives a
+    /// notification and the required condition is met. There are no spurious
+    /// wakeups when calling this.
```

Comment:
Saying that there aren't any spurious wakeups doesn't seem totally accurate - the thread is awake when the closure is executing.

Reply:
You're right.  I meant to say this function won't return due to spurious wakes.  I'll clarify.